org.OpenT2T.Sample.SuperPopular.Smoke/Wink Alarm/js/manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <!-- associated voice handler -->
+  <voice id="org.OpenT2T.Voice.Cortana.Vcd.Sample.SuperPopular.LightControl" />
+  <!-- associated schema -->
+  <schema id="org.OpenT2T.Sample.SuperPopular.Smoke" />
```

Comment:
I prefer org.OpenT2T.Sample.SuperPopular.SmokeDetector for the schema name. What do you think? "Smoke" by itself seems to not be enough.